### PR TITLE
#50 docs extend installation info

### PR DESCRIPTION
**Pull request by luisalbaceteinbibo**
### Description of this issue.
Extended the explanation on how to configure Maya environment variables with screenshots at Linux and Windows.

### Results
- Preview page: https://inbibo.co.uk/docs/preview?sheet_url=https%3A%2F%2Fgithub.com%2FInbibo%2Fadonisfx_docs%2Fblob%2F%252350-docs-extendInstallationInfo%2Fdocs%2Finstallation.md
- MD file to copy to preview page: https://github.com/Inbibo/adonisfx_docs/blob/%2350-docs-extendInstallationInfo/docs/installation.md

Issue of this PR: #50

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -5,19 +5,70 @@
...
+
+AdonisFX is distributed for Maya 2022, 2023 and 2024. If you have multiple Maya versions installed, make sure to [download](https://inbibo.co.uk/adonisfx/downloads) the right AdonisFX build and configure the environment to point to the right AdonisFX version. Remind that multiple versions of AdonisFX can be installed, but only one can be loaded at a time.
+
+There are two different methods of configuring this variable: Setting the value in the `Maya.env` config file; or setting the value in the system to be permanent. In the following sections we explain these two methods for Windows and Linux.
+
+### Configure Environment On Windows
+
+#### Method 1: Maya.env
+
+The Maya.env file is version dependent. It means that every instance of Maya will evaluate the environment file associated to its version. The default location of this file is `DRIVE:/Users/username/Documents/maya/%MAYA_VERSION%`.
```
Comment (lucasferryinbibo) [inline docs/installation.md:22]:
```suggestion
The `Maya.env` file is version dependent. It means that every instance of Maya will evaluate the environment file associated to its version. The default location of this file is `DRIVE:/Users/username/Documents/maya/%MAYA_VERSION%`.
```

File: docs/installation.md [hunk 2]
```diff
@@ -5,19 +5,70 @@
...
+
+> ![Windows "Maya.env" file location.](/images/windows_maya_env_file_location.png)
+> <b>Figure 1</b>: Location of "Maya.env" file in Windows. Example for Maya 2024.
+
+Add this line making sure to specify the folder containing the AdonisFX.mod file and the module will be loaded the next time you launch Maya:
```
Comment (lucasferryinbibo) [inline docs/installation.md:27]:
```suggestion
Add this line making sure to specify the folder containing the `AdonisFX.mod` file and the module will be loaded the next time you launch Maya:
```

File: docs/installation.md [hunk 3]
```diff
@@ -5,19 +5,70 @@
...
+### Configure Environment On Linux
+
+#### Method 1: Maya.env
+
+The Maya.env file is version dependent. It means that every instance of Maya will evaluate the environment file associated to its version. The default location of this file is `~/maya/$MAYA_VERSION`.
+
+> ![Linux "Maya.env" file location.](/images/linux_maya_env_file_location.png)
+> <b>Figure 5</b>: Location of "Maya.env" in Linux. Example for Maya 2022.
+
+Add this line making sure to specify the folder containing the AdonisFX.mod file and the module will be loaded the next time you launch Maya:
```
Comment (lucasferryinbibo) [inline docs/installation.md:57]:
```suggestion
Add this line making sure to specify the folder containing the `AdonisFX.mod` file and the module will be loaded the next time you launch Maya:
```

File: docs/installation.md [hunk 4]
```diff
@@ -5,19 +5,70 @@
...
+
+> ![Windows system properties.](/images/windows_system_properties.png)
+> <b>Figure 2</b>: System Properties window.
+
+2. Click on *Environment Variables* and a new window will pop showing all the environment variables configured at the system level and for the current user.
+
+> ![Windows environment variables.](/images/windows_environment_variables.png)
+> <b>Figure 3</b>: Environment variables of the system.
+
+3. Click on *New...* button for the level of you prefer (system or user) and configure `MAYA_MODULE_PATH` with the path containing the AdonisFX.mod file.
```
Comment (lucasferryinbibo) [inline docs/installation.md:43]:
```suggestion
3. Click on *New...* button for the level of you prefer (system or user) and configure `MAYA_MODULE_PATH` with the path containing the `AdonisFX.mod` file.
```

**Review by lucasferryinbibo — APPROVED** [review]
Some very small comments. Overall very clear and well explained!

File: docs/installation.md (hunk 1 above)
Comment (luisalbaceteinbibo) [reply docs/installation.md]: Done

File: docs/installation.md (hunk 2 above)
Comment (luisalbaceteinbibo) [reply docs/installation.md]: Done

File: docs/installation.md (hunk 3 above)
Comment (luisalbaceteinbibo) [reply docs/installation.md]: Done

File: docs/installation.md (hunk 4 above)
Comment (luisalbaceteinbibo) [reply docs/installation.md]: Done

**Review by carlosmonteagudoinbibo — APPROVED** [review]

**Review by enriquemeseguerinbibo — APPROVED** [review]

**Pull request merged**